sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md

```diff
@@ -10,6 +10,16 @@
 
 ### Other Changes
 
+## 1.2.2 (2025-04-24)
+
+### Features Added
+
+- Upgraded api-version tag from 'package-2023-06' to 'package-2025-02'. Tag detail available at https://github.com/Azure/azure-rest-api-specs/blob/97ee23a6db6078abcbec7b75bf9af8c503e9bb8b/specification/recoveryservicesbackup/resource-manager/readme.md.
+- Support for SAPAse Database.
+- Added MUA support for new operations.
+- Added recovery point tier information for AFS.
+- Added support for undelete softdeleted container.
```

Comment:
I think you added this because you would like to release a new stable version.
This now has some issues.
First of all, if we want a stable release, considering your changes contain an api-version bump, the new version should be `1.3.0`. Usually when the new change contain new API, we should release a new minor version instead of a patch version.

Therefore to fix this, you need to remove the `1.3.0-beta.1 (Unreleased)` section (and all its subsections, and also please update the version number in csproj file (this is the reason you see the CI fails - the CI requires the latest version in changelog must be the same as the version in csproj).

Reply:
Have fixed it as suggested.